src/modules/drawerDetails.js

```diff
@@ -0,0 +1,42 @@
+import Details from './details';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const DrawerDetails = (props) => {
+    const {
+        bleed,
+        className,
+        color,
+        columnProps,
+        columns,
+        data,
+        style,
+    } = props;
+
+    return (
+        <React.Fragment>
+            <Details
+                bleed={bleed}
+                className={className}
+                color={color}
+                columnProps={columnProps}
+                columns={columns}
+                data={data}
+                style={style}
+                moduleType="drawer"
+            />
+        </React.Fragment>
```

Comment:
No need for `<React.Fragment />`, let's remove. Also, let's spread the passed in props instead of listing them this way. 

e.g. 
```
<Details
    {...props}
    moduleType="drawer"
/>
```

Reply:
Fixed